### DISCUSSION TITLE
part 2, empates de double-sort

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#117] [Tarea 1] [answered]
Hola! En casos de empate en los dos criterios (age y membership), se debe ordenar por id? en ese caso, el orden de id es de menor a mayor siempre? o debe seguir el order1? o el order2?

**Comment by Segismundios** [comment, marked as answer]
Hola @GuilleStanke. En caso de que haya un empate entre los pingüinos, se debe respetar el orden de ingreso, es decir, aquellos que ingresaron primero deben quedar antes en el orden.
Saludos!
